wurst/Demo.wurst

```diff
@@ -95,5 +95,5 @@ init
 
     EventListener.add(EVENT_PLAYER_UNIT_DEATH) ->
         let u = EventData.getTriggerUnit()
-        destroy networkHash.get(u)
-        networkHash.remove(u)
\ No newline at end of file
+        if networkHash.has(u)
+            destroy networkHash.getAndRemove(u)
```

Comment:
Drive-by fix, because networkHash only has farms. If you kill another unit this causes stacktrace